packages/engine/Specs/DataSources/CallbackPropertySpec.js

```diff
@@ -1,7 +1,10 @@
 import { JulianDate, CallbackProperty } from "../../index.js";
 
 describe("DataSources/CallbackProperty", function () {
-  const time = JulianDate.now();
+  let time;
+  beforeEach(function () {
+    time = JulianDate.now();
+  });
```

Comment:
This change is unrelated, but I noticed it was interfering with other specs on a test failure.